docs/contribute/whiteboard.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Saros HTML Whiteboard
+---
+
+# {{ page.title }}
+
+The Saros Whiteboard was designed to enable Saros users to communicate ideas graphically.
+
+## Developing the Whiteboard
+
+### Setup
+Use the command `npm run setup`, note that `npm install` does not suffice.
+
+### Build for Development
```

Comment:
As far as I know it is necessary (as long as the HTML WB is not default) to uncomment the correponding view in the `plugin.xml`.

Reply:
Are there further steps to do? I was able to access the new WB, but nothing was synced between the participants.
I did the following:
* `npm run setup`, `npm run build`
* uncomment the new view
* start ALICE via the run configuration
* start BOB via the run configuration
* Connection with the xmpp account of alice, same for bob
* Start session with one file
* Open WB on ALICEs side, same for bob
* Draw something on ALICEs side
* Nothing happens on BOBs side (and vice versa)

Did I missed something?

Reply:
The problem was actually not in the setup process, there were missing dependencies (#423)

Reply:
I have updated the file to include un-commenting the view